crates/uv-cli/src/lib.rs

```diff
@@ -427,6 +427,10 @@ pub enum SelfCommand {
 pub struct SelfUpdateArgs {
     /// Update to the specified version. If not provided, uv will update to the latest version.
     pub target_version: Option<String>,
+
+    /// Specify a github token for authentication during the update process.
```

Comment:
Maybe something like
```suggestion
    /// A GItHub token for authentication.
    ///
    /// A token is not required but can be used to reduce the chance of encountering rate limits.
```

Reply:
You'll need to run `cargo dev generate-all` to update the reference docs.

Reply:
> You'll need to run `cargo dev generate-all` to update the reference docs.

Run but nothing is updated.

Reply:
Oh sorry, we don't have this command in the CLI reference — which is a separate problem, I'll fix that.